### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by mumbereh**
I have added the form to my mobile portfolio project. 
Making it complete.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
                   </li>
               </ul>
              </section>
+
+             <section class="form">
+             <form action="" method="post">
+                <h2 class="form-header">
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As required for the project, kindly make sure your Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
 * `action` value should be similar to `https://formspree.io/f/{form_id}`
 * `method` value equals `POST`

Comment (tresorsawasawa) [inline index.html]: - [ ] Once you have fixes the above change, kindly make sure you check the `network` tab in your browser dev tools to confirm that the form is submitted successfully with valid input values. Below is how your application looks after a user has submitted a form:
Screenshot from 2023-03-28 17-24-29

File: index.html [hunk 2]
```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
...
+                <ul class="ul_form">
+                    <li>
+                        <input type="text" maxlength="30" name="name" placeholder="enter your name......" required>
+                    </li>
+                    <li>
+                        <input type="text" name="email" placeholder="enter your email......"required>
+                    </li>
+                    <li>
+                        <textarea name="message" maxlength="500" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                    </li>
```
Comment (tresorsawasawa) [inline index.html:187]: - [ ] As required for the project, kindly make sure all your input fields have the `required` attribute to activate the validation on submit. I noticed that you forgot add the `required` attribute in the `textarea` input

File: style.css [hunk 3]
```diff
@@ -409,3 +409,98 @@ li {
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+form p {
```
Comment (tresorsawasawa) [inline style.css]: - [ ] As required for the project, kindly make sure `Flexbox` is used to layout the elements of your footer section.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @mumbereh and @Mar12358 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
...
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!
+                </p>
+                <ul class="ul_form">
+                    <li>
+                        <input type="text" maxlength="30" name="name" placeholder="enter your name......" required>
+                    </li>
+                    <li>
+                        <input type="text" name="email" placeholder="enter your email......"required>
```
Comment (victorokoroji) [inline index.html:183]: [OPTIONAL]

- Kindly ensure that your email input fields have the correct type attribute i.e email input's `type` equals `email` which is the proper way of declaring an input field 👍🏽

File: index.html [hunk 5]
```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
...
+                <h2 class="form-header">
+                    Contact me
+                </h2>
+                <p>
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!
+                </p>
+                <ul class="ul_form">
+                    <li>
```
Comment (victorokoroji) [inline index.html]: - [ ] As suggested by the previous reviewer, kindly make sure Flexbox is used to layout the elements of your contact section and the form itself so as to make the section responsive and also gives you the added advantage of laying out the elements the way you want to improve user experience 👍🏽 .

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @mumbereh, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -409,3 +409,109 @@ li {
...
+  padding-right: 24px;
+  display: flex;
+  justify-content: center;
+}
+
+.form-body {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 4%;
+}
```
Comment (Olamarx) [inline style.css:455]: Nice one using flex 👍

**Review by Olamarx — APPROVED** [review]
Hi @mumbereh,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Flexbox ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**